Update django-contrib-comments

vkaracic opened the pull request:
Version 1.8.0 of django-contrib-comments moved away from
using the SITE_ID settings attribute in favor of supporting
multi-tenancy.

LEARNER-1583 + EDUCATOR-733

@edx/helio and @awais786 please review.

vkaracic commented:
@clintonb test added!

clintonb commented on course_discovery/apps/publisher_comments/tests/test_forms.py:
```diff
@@ -0,0 +1,36 @@
+
+
+class CommentsFormTests(TestCase):
+    """Tests for the CommentsForm form."""
```
I've always found these class-level comments worthless. Feel free to remove this.

clintonb commented on course_discovery/apps/publisher_comments/tests/test_forms.py:
```diff
@@ -0,0 +1,36 @@
+        return data
+
+    def test_get_comment_model(self):
+        """Retrieves the Comments model."""
```
What retrieves the model? This doc string doesn't really help me understand what you're testing or why.

clintonb commented on course_discovery/apps/publisher_comments/tests/test_forms.py:
```diff
@@ -0,0 +1,36 @@
+        self.assertEqual(model, Comments)
+
+    def test_get_comment_create_data(self):
+        """Sets the right site_id value."""
```
Same as above. Help me understand why this test is useful.

vkaracic commented:
@clintonb I have somewhat addressed your comments. I've created a test on a more suitable place. The error that happens when upgrading this package happens when data is posted the comment post endpoint hence this test. And I have confirmed that when I remove the `site_id` parameter this new test fails.
